### PR TITLE
Update ATI ContentId logic for CPS assets 

### LINKED ISSUE #9852: Update ATI ContentId logic for CPS assets
**Is your feature request related to a problem? Please describe.**
We currently construct the ContentId in Simorgh.  Ares now provides the value for us.  We can simplify Simorgh by using the value provided by Ares.

**Describe the solution you'd like**
Ares have updated their output for both CPS and Optimo assets to explicitly expose the ContentId value to send to ATI.

We should remove the existing logic in Simorgh which generates this, and replace the value with the one provided by Ares as metadata.analyticsLabels.contentId.  This should apply to both AMP and Canonical.

We have already done this for Optimo pages see: https://github.com/bbc/simorgh/pull/9717
we now want to rationalise the logic for CPS pages - PR has been raised for this previously see https://github.com/bbc/simorgh/pull/9684

Current CPS implementation: https://github.com/bbc/simorgh/blob/538a49c1b7368c708e4ddfd61c31d44db63f9f5b/src/app/lib/analyticsUtils/index.js#L223-L228

This change means we no longer need separate treatment for CPS and Optimo asset here, nor are we constructing the value for CPS assets.

Note that for CPS assets this will result in "curie:asset:" being added additionally in the content id, compared to what we have now. This is expected, and standardises the contentId value with that used in other systems such as Passports and SCV.

Similarly, for Optimo this will result in "asset:" being added additionally in the content id, compared to what we have now.  This is also expected, for the same reasons.

# Examples
## CPS

Ares: https://ares-api.api.bbci.co.uk/api/asset/news/science-environment-59022676
FABL: https://fabl.api.bbci.co.uk/module/article?uri=/news/science-environment-59022676
### Ares data
```
{
  "metadata": {
    "analyticsLabels": {
      "contentId": "urn:bbc:cps:curie:asset:29375628-9511-42e6-be88-ebaa1158f597",
```

**Describe alternatives you've considered**
Update logic in Simorgh to construct URNs matching those already provided by Ares.  This seems pointless duplication and fragile.

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by emilysaffron**
Resolves #9852

**Overall change:**
Previously we were building the contentId string and now Ares provides a value which the code just reads in.

**Code changes:**

- Refactored getContentId function to get the contentId from the analyticsLabels property
- Removed a test case that is no longer relevant.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: data/gahuza/legacyAssets/video/2016/01/160108_australia_fire_video.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     },
     "analyticsLabels": {
       "cps_asset_type": "map",
-      "cps_asset_id": "42074194"
+      "cps_asset_id": "42074194",
```
Comment (jimjohnsonrollings) [inline data/gahuza/legacyAssets/video/2016/01/160108_australia_fire_video.json:25]: I know this hasn't been changed, and it's only fixture data... but this appears to be a CPS Asset ID in a TopCat asset, which I don't think would ever be a thing in reality.

Comment (MuhammadAHussain) [reply data/gahuza/legacyAssets/video/2016/01/160108_australia_fire_video.json:25]: Fixture seems to match the asset on live; think it's the way the asset it.

Comment (jimjohnsonrollings) [reply data/gahuza/legacyAssets/video/2016/01/160108_australia_fire_video.json:25]: So it does.  Okay, no worries then :)

**Review by HarveyPeachey — DISMISSED** [review]
Nice work!

**Review by ryanmccombe — DISMISSED** [review]

File: src/app/lib/analyticsUtils/index.js [hunk 2]
```diff
@@ -227,12 +227,11 @@ export const getPublishedDatetime = (attribute, data) => {
     : null;
 };
 
-export const getContentId = (assetType, pageData) => {
-  const curie = path(['metadata', 'locators', 'curie'], pageData);
-  const guid = curie.split('/').pop();
-  const contentId = `urn:bbc:${assetType}:`.concat(guid);
-  return contentId;
-};
+export const getContentId = pathOr(null, [
```
Comment (ryanmccombe) [inline src/app/lib/analyticsUtils/index.js:230]: Nice one 👍  not many people notice these ramda functions can be curried!

**Review by HarveyPeachey — APPROVED** [review]
LGTM, looks like e2e are failing, think it's to do with the ongoing content store issue so we good

**Review by amoore108 — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Testing:

Checked few Articles on different services for eg;

* http://localhost:7080/igbo/articles/cr1lw620ygjo.json
* http://localhost:7080/russian/articles/ck7pz7re3zgo.json

Checked page looks ok when there is no contentid.

LGTM..

**Pull request merged**